Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Compact/BasicOperators/CompactNegativeNodeable.ScriptCanvasNodeable.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScriptCanvas Include="Include/ScriptCanvas/Libraries/Compact/BasicOperators/CompactNegativeNodeable.h" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+			  xsi:noNamespaceSchemaLocation="../../../AutoGen/ScriptCanvasNodeable.xsd">
+    <Class Name="CompactNegativeNodeable"
```

Comment:
Is this the right name of this node? It may imply that f(-1) = -1, since the number is still negative. Maybe the name should be `CompactNegateNumberNodeable` ?

Reply:
Hmm yeah I see what you are saying, but I feel like `CompactNegateNumberNodeable` could imply the same thing

Reply:
Though, I think I can see how it might make it a bit clearer to the user, I'll change that